### PR TITLE
docs align and fixes

### PR DESCRIPTION
**Pull request by fabianflx**

### CODE REVIEW

File: lib/elrondex/account.ex [hunk 1]
```diff
@@ -59,6 +60,10 @@ defmodule Elrondex.Account do
       ...> |> Elrondex.Account.from_private_key()
       ...> |> Map.get(:address)
       "erd1edmdkecu95u6aj9ehd0lf3d97qw85k86pkqqdu5029zcydslg7qs3tdc59"
+
+      iex> Elrondex.Test.Bob.hex_private_key()
+      ...> |> Elrondex.Account.private_key_to_address()
```
Comment (elrondex) [inline lib/elrondex/account.ex]: from_private_key

File: lib/elrondex/transaction.ex [hunk 2]
```diff
@@ -202,10 +202,10 @@ defmodule Elrondex.Transaction do
   def is_required_sign_field(_), do: false
 
   @doc """
-  Returns the non-empty values of a transaction in jSON format
+  Returns raw data to sign JSON format.
```
Comment (savuvictor) [inline lib/elrondex/transaction.ex:205]: Returns raw data to sign in JSON format.

Comment (victorflx) [inline lib/elrondex/transaction.ex]: Returns raw data to sign in JSON format.

File: lib/elrondex/account.ex [hunk 3]
```diff
@@ -59,6 +60,10 @@ defmodule Elrondex.Account do
       ...> |> Elrondex.Account.from_private_key()
       ...> |> Map.get(:address)
       "erd1edmdkecu95u6aj9ehd0lf3d97qw85k86pkqqdu5029zcydslg7qs3tdc59"
+
+      iex> Elrondex.Test.Bob.hex_private_key()
+      ...> |> Elrondex.Account.from_private_key()
+      "erd1edmdkecu95u6aj9ehd0lf3d97qw85k86pkqqdu5029zcydslg7qs3tdc59"
```
Comment (victorflx) [inline lib/elrondex/account.ex:66]: |> Map.get(:private_key)
Run doctests, check if its ok.

File: lib/elrondex/transaction.ex [hunk 4]
```diff
@@ -136,19 +136,6 @@ defmodule Elrondex.Transaction do
     end
   end
 
-  def prepare_network(%Transaction{} = tr, network) do
```
Comment (victorflx) [inline lib/elrondex/transaction.ex:139]: align with main, why does this disappears?

File: lib/elrondex/transaction.ex [hunk 5]
```diff
@@ -136,19 +136,6 @@ defmodule Elrondex.Transaction do
     end
   end
```
Comment (victorflx) [inline lib/elrondex/transaction.ex:138]: align with main, why does this disappears?

**Pull request closed without merging**